compose.yaml

```diff
@@ -5,9 +5,9 @@ services:
       - "3000:3000"
     volumes:
       - ./container/:/var/lib/bowtie
+    env_file: config.env
```

Comment:
Didn't know it could do this; nice

Reply:
yeah finch won't work without this since it can't read the env